doc/sphinx/language/cic.rst

```diff
@@ -1141,6 +1142,26 @@ Conversion is preserved as any (partial) instance :math:`I_j~q_1 … q_r` or
 :math:`C_i~q_1 … q_r` is mapped to the names chosen in the specific instance of
 :math:`\ind{p}{Γ_I}{Γ_C}`.
 
+.. warning::
+
+   The restriction that sorts are introduced by the inductive
+   declaration prevents inductive types declared in sections to
+   be template-polymorphic: they cannot parameterize over their
```

Comment:
This is not specific enough apparently, should be:

>prevents inductive types declared in sections to be template-polymorphic on section universes

or something along those lines.